test/test-app/vendor/kubeflow/core@f7a68336ad7a65c2cbba8462e89d24a10626687e/tf-job-operator.libsonnet

```diff
@@ -102,7 +102,7 @@
             containers: [
               {
                 command: [
-                  "/opt/kubeflow/tf-operator.v1",
+                  "/opt/tf-operator.v1",
```

Comment:
Do you really want to change the directory structure? This has to  be reflected in manifests also

Reply:
The challenge here is that distroless has no method of creating a directory since it does not have a shell within the image.
The idea with distroless is just to place a binary within the image and that should be the only step.